data-service/src/main/java/com/amazonaws/blox/dataservice/model/Environment.java

```diff
@@ -31,16 +31,14 @@
 
   @NonNull private String environmentId;
```

Comment:
Since this is part of the API, and not the data model, could we expose this as `(accountId, environmentName)` instead? Otherwise we have to spread ID construction/deconstruction logic around the frontend too.